ui/src/components/slider/use-slider.js

```diff
@@ -235,10 +235,10 @@ export default function ({ updateValue, updatePosition, getDragging, formAttrs }
 
   function convertRatioToModel (ratio) {
     const { min, max, step } = props
-    let model = min + ratio * (max - min)
+    let model = innerMin.value + (ratio - innerMinRatio.value) * (max - min)
```

Comment:
Why are you doing this?

Reply:
Because otherwise it wouldn't entirely fix the issue?.. 

The calculation will return the same result before and after, but this avoid floating point operation issues, if we have a float as min and a ratio also as float we end up with a float that is not precise enough eg 3.999999
So by using innerMin directly and subtracting it from the ratio which turns up exactly the same in arithmetic we can avoid the floating point precision issue